biogrid/libs/biogrid-simulator/src/lib/config/bio-constants.ts

```diff
@@ -30,6 +30,21 @@ export const enum GRID_ITEM_NAMES {
   GRID = 'grid',
 };
 
+export const RESISTANCE = {
```

Comment:
Exported items should have jsdocs.  This would be a good place to note the units if all of the resistances have the same units

Reply:
Alright, I will change that asap

Reply:
I will add it in JSdocs though and explain everything in there